.eslintrc.js

```diff
@@ -1,40 +1,69 @@
+/** @format */
+
 module.exports = {
 	root: true,
-	'extends': [
-		'wpcalypso/react',
-		'plugin:jsx-a11y/recommended',
-		'plugin:jest/recommended',
-	],
+	extends: [ 'wpcalypso/react', 'plugin:jsx-a11y/recommended', 'plugin:jest/recommended' ],
 	parser: 'babel-eslint',
 	env: {
 		browser: true,
 		'jest/globals': true,
+		// mocha is only still on because we have not finished porting all of our tests to jest's syntax
 		mocha: true,
-		node: true
+		node: true,
 	},
 	globals: {
+		// this is our custom function that's transformed by babel into either a dynamic import or a normal require
 		asyncRequire: true,
+		// this is the name of the project from the build config. Injected at boot in a script tag.
 		PROJECT_NAME: true,
+		// this is the SHA of the current commit. Injected at boot in a script tag.
 		COMMIT_SHA: true,
 	},
-	plugins: [
-		'jest',
-		'jsx-a11y',
-		'import',
-	],
+	plugins: [ 'jest', 'jsx-a11y', 'import' ],
 	rules: {
-		camelcase: 0, // REST API objects include underscores
+		// REST API objects include underscores
+		camelcase: 0,
+
+		// TODO: why did we turn this off?
```

Comment:
I think it errors because there are still `chai` matchers which aren't compatible with Jest syntax.